node/src/Pos/Wallet/Web/ClientTypes/Functions.hs

```diff
@@ -149,10 +149,12 @@ mkCTx diff th@THEntry {..} meta pc wAddrMetas = do
     return CTx {..}
   where
     ctId = txIdToCTxId _thTxId
+    encodeTxOut :: TxOut -> (CId Addr, CCoin)
+    encodeTxOut TxOut{..} = (addressToCId txOutAddress, mkCCoin txOutValue)
     inputs = _thInputs
     outputs = toList $ _txOutputs _thTx
-    ctInputAddrs = ordNub $ map addressToCId (_thInputAddrs th)
-    ctOutputAddrs = map addressToCId _thOutputAddrs
+    ctInputs = map encodeTxOut _thInputs
```

Comment:
We probably should merge inputs with equal addresses here.

Reply:
@Martoon-00 was this resolved ? (there were no commits after this one)

Reply:
It was, commit is below